P5/Source/Specs/specGrp.xml

```diff
@@ -62,11 +62,21 @@ contains specifications for the elements
   <exemplum versionDate="2008-04-06" xml:lang="fr">
```

Comment:
The `@versionDate` attribute needs to be set to whatever day you changed the example, @rettinghaus (or today if you don’t remember).

Reply:
done.